components/smee/base/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
(optional) I assume you are provisioning for production and staging using the same resources eventually but currently this file is not used, you could remove the base folder. Another option would be to move all the resources from staging folder to the base folder and include base in staging kustomize file.